src/plugin.js

```diff
@@ -183,30 +209,31 @@ export default class TagWrangler extends Plugin {
             search = searchPlugin && searchPlugin.instance,
             query = search && search.getGlobalSearchQuery(),
             random = this.app.plugins.plugins["smart-random-note"],
-            menu = e.obsidian_contextmenu.addItem(item("pencil", "Rename #"+tagName, () => this.rename(tagName)));
+            menu = e.obsidian_contextmenu.addItem(item("pencil", `${this.it.Rename} #`+tagName, () => this.rename(tagName)));
 
         menu.addSeparator();
         if (tagPage) {
             menu.addItem(
-                item("popup-open", "Open tag page", (e) => this.openTagPage(tagPage, false, Keymap.isModEvent(e)))
+                item("popup-open", this.it[OPEN_TAG_PAGE], (e) => this.openTagPage(tagPage, false, Keymap.isModEvent(e)))
             )
         } else {
             menu.addItem(
-                item("create-new", "Create tag page", (e) => this.createTagPage(tagName, Keymap.isModEvent(e)))
+                item("create-new",this.it[CREATE_A_TAG], (e) => this.createTagPage(tagName, Keymap.isModEvent(e)))
             )
         }
 
         if (search) {
             menu.addSeparator().addItem(
-                item("magnifying-glass", "New search for #"+tagName, () => search.openGlobalSearch("tag:" + tagName))
+                item("magnifying-glass", `${this.it[NEW_SEARCH_FOR]} #`+tagName, () => search.openGlobalSearch("tag:" + tagName))
             );
             if (query) {
                 menu.addItem(
-                    item("sheets-in-box", "Require #"+tagName+" in search"  , () => search.openGlobalSearch(query+" tag:"  + tagName))
+                    item("sheets-in-box", `${this.it[REQUIRE]} #${tagName} ${this.it[IN_SEARCH]}`, 
+                    () => search.openGlobalSearch(query+" tag:"  + tagName))
                 );
             }
             menu.addItem(
-                item("crossed-star" , "Exclude #"+tagName+" from search", () => search.openGlobalSearch(query+" -tag:" + tagName))
+                item("crossed-star" , `${this.it[EXCLUDE]} #${tagName} ${this.it[FROM_SEARCH]}`, () => search.openGlobalSearch(query+" -tag:" + tagName))
```

Comment:
In short, this violates the very basics of translation.   

One must NEVER concatenate strings for any reason.

Word order varies by language.  Words need to conjugation to agree with substituted elements (gender agreement, case agreement (declination), sound-agreement (different words used depending if e/i or o/u are in the object, etc).  

This fails for languages that:
- Place Verb Last (JP) - `#tag from-search exclude`
- Require conjugation of tag (FI) `exclude #tag":n" from-search`
- Place Complement First - `from-search exclude #tag`
- That split the components - `search #tag-target-from exclude`
- That require reduplication - `#tag-from-search #tag-target exclude`
Etc.